values.yaml

```diff
@@ -219,6 +219,28 @@ global:
     # Requires consul-k8s 0.15.0+.
     createFederationSecret: false
 
+  # Resource settings for lifecycle-sidecar containers
```

Comment:
Can you explain a bit what the lifecycle sidecar is and where it is used, i.e. it ensures Consul services are always registered with their local consul clients and it's used in mesh, ingress and terminating gateways along with every connect-injected service.